analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/data/MavenCoordinateTest.java

```diff
@@ -1,115 +0,0 @@
-/*
```

Comment:
Why has this file been deleted? It is just in the wrong place... Instead of deleting it, it should be moved to FASTEN core (where the Maven Coordinate is located).

Reply:
Sorry, forgot to push the tests for them in core. It's here a0ba202.